### PR TITLE
HDDS-9220. Let ContainerBalancerConfiguration#toString print more info

### PR DESCRIPTION
**Pull request by YuanbenWang**
## What changes were proposed in this pull request?

This pr aims to let ContainerBalancerConfiguration#toString print more information when called.

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-9220

## How was this patch tested?

From:
Before

To:
After

### CODE REVIEW

**Review by siddhantsangwan — COMMENTED** [review]
@YuanbenWang thanks for working on this. I just have one comment below.

File: hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/balancer/ContainerBalancerConfiguration.java [hunk 1]
```diff
@@ -411,15 +411,42 @@ public String toString() {
...
         "Max Size Entering Target per Iteration",
         maxSizeEnteringTarget / OzoneConsts.GB,
         "Max Size Leaving Source per Iteration",
-        maxSizeLeavingSource / OzoneConsts.GB);
+        maxSizeLeavingSource / OzoneConsts.GB,
+        "Number of Iterations",
+        iterations,
+        "Time Limit for Single Container's Movement",
+        Duration.ofMillis(moveTimeout).toMinutes(),
+        "Time Limit for Container Replication's Move",
```
[the remark this answers is not in the document]
Comment (YuanbenWang) [reply hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/balancer/ContainerBalancerConfiguration.java]: Thank you for the review~! I have changed it.

**Review by Xushaohong — COMMENTED** [review]
Giving more direct config details. It looks good to me. thx @YuanbenWang

**Comment by YuanbenWang** [comment]
> Giving more direct config details. It looks good to me. thx @YuanbenWang

Thanks for the review~!

**Review by dineshchitlangia — APPROVED** [review]
+1 pending CI

**Comment by YuanbenWang** [comment]
> +1 pending CI

Thanks for the review~! I have fix the checkstyle.

**Comment by YuanbenWang** [comment]
@dineshchitlangia CI got 503 error code, so could you help to retrigger it ? Thx.

**Comment by YuanbenWang** [comment]
In my own CI, all test are success.
HDDS-9220

**Comment by siddhantsangwan** [comment]
I've triggered a CI run.

**Comment by YuanbenWang** [comment]
> I've triggered a CI run.
All the test are success. Would you please help merge this pr?

**Review by ChenSammi — APPROVED** [review]

**Comment by ChenSammi** [comment]
Thanks @YuanbenWang for the contribution, @Xushaohong and @dineshchitlangia, @siddhantsangwan for the code review.

**Comment by YuanbenWang** [comment]
@siddhantsangwan @ChenSammi @dineshchitlangia @Xushaohong Thank you for your assistance in reviewing the code!

**Pull request merged**